src/webgl/context-features.js

```diff
@@ -80,7 +78,7 @@ export function hasFeature(gl, feature) {
 export function hasFeatures(gl, features) {
   features = Array.isArray(features) ? features : [features];
   return features.every(feature => {
-    return getFeature(gl, feature) || gl.getExtension(feature);
```

Comment:
@ibgreen , this would always return true since , gl.getExtension would return an object. Given we are also checking gl.getExtension in getFeature, removing it here.